### ISSUE TITLE
[Live Components]Docs for Data Binding are insufficient

### ISSUE THREAD

**Issue by truckee** [symfony/ux#457]
Attempts to reproduce the behavior demonstrated at [Data Binding](https://symfony.com/bundles/ux-live-component/current/index.html#liveprops-stateful-component-properties) fail. The docs appear to be insufficient. For example,
```
<div {{ attributes }}>
    <input
        type="number"
        value="{{ max }}"
        data-model="max"
    >

    Generating a number between 9 and {{ max }}
    <strong>{{ this.randomNumber }}</strong>
</div>
```
lacks `<form>...</form>` and `<button>...</button>`. But even when added the value of `max` always reverts to the value in `.../Components/RandomNumberComponent`. Surely something else is missing.

**Comment by kbond** [comment]
Hey @truckee, are you manually refreshing the page? Does your `$max` property have `#[LiveProp(writable: true)]`? The `<strong>` value should be changing as you type into the input.

**Comment by truckee** [comment]
Hiya.

... are you manually refreshing the page? If you mean after updating the source code, yes.
Does your $max property have #[LiveProp(writable: true)]? Yes.

Puzzle pieces:
```
#[AsLiveComponent('random_number')]
class RandomNumberComponent
{
    use DefaultActionTrait;

    #[LiveProp(writable: true)]
    public int $max = 1000;

    public function getRandomNumber(): int
    {
        return rand(0, $this->max);
    }
}
```
and
```
<div {{ attributes }}>
    <form>
        <input
            type="number"
            value="{{ max }}"
            data-model="max"
            >    
        <button data-action="live#$render">Generate a new number!</button>
    </form>
    <strong>{{ this.randomNumber }}</strong>
</div>
```
and
```
{{ component('random_number') }}
```

**Comment by kbond** [comment]
I'm not terribly proficient with the frontend part of live components. @weaverryan, any ideas?

**Comment by truckee** [comment]
I misunderstood the intended behavior. I thought that changing the value of max at input would cause the value of `max` to change for the _next_ **Generate a new number**. That is not what I observed. Only today did I note what must be the intended behavior: when the input value of `max` is changed a new random number within the revised bounds **immediately** appears. This behavior is substantially more useful.

Edit: For whatever reason, using the Symfony cli server did NOT show the intended behavior. Only when I used my Apache server did the intended behavior occur. 

Apologies for my misunderstanding.
